geode-core/src/main/java/org/apache/geode/internal/cache/event/EventSequenceNumberHolder.java

```diff
@@ -99,12 +114,26 @@ public String toString() {
   public void fromData(DataInput in) throws IOException, ClassNotFoundException {
     lastSequenceNumber = in.readLong();
     versionTag = (VersionTag) DataSerializer.readObject(in);
+    if (StaticSerialization.getVersionForDataStream(in).isNotOlderThan(KnownVersion.GEODE_1_15_0)) {
```

Comment:
This is ok but sometimes it's easier to read if you import static utility methods and constants:
```
import static org.apache.geode.internal.serialization.KnownVersion.GEODE_1_15_0;
import static org.apache.geode.internal.serialization.StaticSerialization.getVersionForDataStream;
```
```
if (getVersionForDataStream(in).isNotOlderThan(GEODE_1_15_0)) {
```